packages/pwa-kit-react-sdk/src/schemas/sdk-config.json

```diff
@@ -0,0 +1,22 @@
+{
+    "definitions": {},
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "pwa-kit-config-schema",
+    "title": "PWA Kit Config Schema",
+    "type": "object",
+    "required": ["url"],
+    "properties": {
+        "url": {
+            "$id": "#pwa-kit-config-schema/url",
+            "title": "url",
+            "type": "object",
+            "required": ["locale"],
+            "properties": {
+                "locale": {
+                    "type": "string",
+                    "enum": ["path", "query_param", "none"]
+                }
+            }
+        }
```

Comment:
This isn't in the SDK yet. I'd be hesitant to be adding this into the SDK before locale support is in the sdk